src/core/storage/Logger.h

```diff
@@ -0,0 +1,25 @@
+#ifndef EUGENE_SRC_EUGENE_API_LOGGER_H
+#define EUGENE_SRC_EUGENE_API_LOGGER_H
+
+#include "spdlog/spdlog.h"
+#include <spdlog/sinks/basic_file_sink.h>
+#include <memory>
```

Comment:
Please separate different header sources. In my opinion keeping `std` headers at the top, then 3-rd party libs and lastly ours works very well.

Reply:
Exactly here I don't need to separate them but I'll do that refactoring to other files

Reply:
:+1: